### PR TITLE
AmazonS3 app

### PR DESCRIPTION
**Pull request by Byte4Craft**
App to manage images via AmazonS3 service

## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVPYTP-62)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/amazons3/models.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+"""
```
Comment (lhalam) [inline eventually/amazons3/models.py]: delete this file because it is not in use

File: eventually/amazons3/urls.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from django.conf.urls import url
+from amazons3.views import ImageManagement
+
+urlpatterns = [
+    url(r'^handle/$', ImageManagement.as_view(), name='handle_image'),
+]
```
Comment (lhalam) [inline eventually/amazons3/urls.py]: add andline

**Review by lhalam — DISMISSED** [review]

File: eventually/amazons3/models.py (hunk 1 above)
Comment (Byte4Craft) [reply eventually/amazons3/models.py]: File deleted

File: eventually/amazons3/urls.py (hunk 2 above)
Comment (Byte4Craft) [reply eventually/amazons3/urls.py]: Added.

**Review by lhalam — APPROVED** [review]

**Pull request merged**